db/models.py

```diff
@@ -1 +1,34 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_TO_CHOOSE = (
+        ("E", "Elf"),
+        ("D", "Dwarf"),
+        ("H", "Human"),
+        ("O", "Ork")
+    )
+    name = models.CharField(max_length=255, unique=True,
+                            choices=RACE_TO_CHOOSE)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The 'bonus' field is defined as a CharField. If this field is intended to store numeric values, consider using an IntegerField or FloatField instead.